cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -181,6 +198,37 @@ public Integer doAction() throws Exception {
     return new Tuple<>(generationInputs, selectedGenerableArtifacts);
   }
 
+  /**
+   * Opens a repeatable prompt with a yes/no question and sets the output directory to the current user directory.
+   *
+   */
+  private void setOutputPathWithPrompt() {
```

Comment:
Cleaner solution to prevent others to program the next yes / no question would be to extract the actual logic from here. Put proper input and output parameters in place and make the question type reusable.

Reply:
Moved the yes no prompt to another method.